website/webpack.config.js

```diff
@@ -13,20 +13,17 @@ const ALIASES = require('ocular-dev-tools/config/ocular.config')({
 // Seems to be a Babel bug
 // https://github.com/babel/babel-loader/issues/149#issuecomment-191991686
 const BABEL_CONFIG = {
-  presets: [
-    'es2015',
-    'stage-2',
-    'react'
-  ].map(name => require.resolve(`babel-preset-${name}`)),
-  plugins: [
-    'transform-decorators-legacy'
-  ].map(name => require.resolve(`babel-plugin-${name}`))
+  presets: ['es2015', 'stage-2', 'react'].map(name => require.resolve(`babel-preset-${name}`)),
```

Comment:
The `map` call is no longer necessary.

Reply:
It is needed website still in babel 6. I can upgrade to 7.